bepasty_cli/cli.py

```diff
@@ -29,6 +36,13 @@
     'MONTHS', 'MONTHS', 'YEARS', 'YEARS', 'FOREVER', 'FOREVER'
 )
 LIFETIME_MAPPING = dict(zip(LIFETIME_CHOICES, LIFETIME_NAMES))
+CONFIG_DEFAULTS = dict(
+    token='',
+    url='',
+    lifetime='1f',
+    insecure=False
+)
+CONFIG_FILENAME = click.get_app_dir('bepasty-client-cli')
```

Comment:
Freedesktop/XDG doesn't want us to use /etc if not necessary, like for programs that are used by multiple users on the same machine or for daemons.